### DISCUSSION TITLE
Config error

### DISCUSSION THREAD

**Discussion by kinnen1988** [jomjol/AI-on-the-edge-device#2996] [General] [answered]
### The Problem

Hallo jomjol, 

ich bekomme alles eingerichtet, bis auf die config für den mqtt. Bei mir erscheint nur:

_
    .__(.)< ( oh oh! This page does not exist! )
    \___)

Vielen Dabk für deinen Support.

### Version

 Firmware version:	Release: v15.7.0 (Commit: 0d0b018+) Firmware build time:	17.02.2024 00:15

### Logfile

```shell
.....
```

**Comment by caco3** [comment, marked as answer]
Funktionieren die anderen Webseiten des Gerätes?
Ev. ist beim Firmware Update etwas fehlgeschlagen, mach einfach nochmals ein OTA update mit der neusten Version.

Wenn das nicht hilf, bitte ein DEBUG Log ab Aufstarten liefern.

**Reply by kinnen1988** [reply]
Vielen Dank, das update hat tatsächlich geholfen, bekomme die Werte jetzt mit mqtt an meinen HA übertragen.
